clients/web/src/utilities/EventStream.js

```diff
@@ -47,12 +47,21 @@ prototype._heartbeat = function () {
 prototype.open = function () {
     if (window.EventSource) {
         var stream = this,
-            url = apiRoot + this.settings.streamPath;
+            url = apiRoot + this.settings.streamPath,
+            params = {};
 
         if (this.settings.timeout) {
-            url += '?timeout=' + this.settings.timeout;
+            params.timeout = this.settings.timeout;
         }
 
+        // Set the "since" argument to filter out notifications
+        // that have already been sent to this client.
+        var since = parseInt(localStorage.getItem('sseTimestamp'));
```

Comment:
Could we refer to this as `window.localStorage`? I know we don't do this for all global symbols, but I think for any that aren't immediately recognizable as browser globals, it helps with readability.